src/Microsoft.OData.Edm/Vocabularies/ValidationVocabularies.xml

```diff
@@ -92,4 +92,18 @@ even though they are defined in an `Orders` entity set.</String>
     <Annotation Term="Core.Description" String="Values are restricted to types that are both identical to or derived from the declared type and a type listed in this collection." />
     <Annotation Term="Core.LongDescription" String="This allows restricting values to certain sub-trees of an inheritance hierarchy. Types listed in this collection that are not derived from the declared type of the annotated model element are ignored." />
   </Term>
+
+  <Term Name="AllowedTerms" Type="Collection(Core.QualifiedTermName)" AppliesTo="Term Property">

```

Comment:
I only see oasis-tc link for MinTerms and MaxTerms, but not this AllowedTerms. Am I missing anything?

Reply:
Yes. That's not in the ODATA-1222. That maybe missed in the original adding validation.

Reply:
This was added in the past few months. See [https://issues.oasis-open.org/browse/ODATA-1212](https://issues.oasis-open.org/browse/ODATA-1212).

---
In reply to: [229765015](https://github.com/OData/odata.net/pull/1310#discussion_r229765015) [](ancestors = 229765015)

Reply:
@mikepizzo Thanks for us know the link.